eclipse/tern.eclipse.ide.server.nodejs.core/src/tern/eclipse/ide/server/nodejs/core/NodejsDebuggersManager.java

```diff
@@ -0,0 +1,47 @@
+/**
+ *  Copyright (c) 2015 Genuitec LLC.
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v1.0
+ *  which accompanies this distribution, and is available at
+ *  http://www.eclipse.org/legal/epl-v10.html
+ *
+ *  Contributors:
+ *  Piotr Tomiak <piotr@genuitec.com> - initial API and implementation
+ */
+package tern.eclipse.ide.server.nodejs.core;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import tern.eclipse.ide.server.nodejs.internal.core.debugging.webclipse.WebclipseDebugger;
+
+public class NodejsDebuggersManager {
+
+	private static Map<String, INodejsDebugger> debuggers = new LinkedHashMap<String, INodejsDebugger>();
+
+	static {
+		// Add supported debuggers
+		addDebugger(new WebclipseDebugger());
```

Comment:
To be clear, I am not against doing the extension point and setting up plugins. It's just that it's quite a lot of work to do it, I don't see any immediate gains and I've already spend some time to get this done. Current structure of code allows for easy refactoring to have an ext. point. It means that if someone needs ext. point, he can contribute.

However, it is important for us to have Webclipse displayed, even if it is not installed. That way people can know that there is something like Webclipse and that it supports debugging of Tern.js.